src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -165,111 +230,50 @@ public boolean publish(SlackRequest slackRequest) {
                         .collect(Collectors.toList())
                 );
             }
+        } catch (IOException e) {
+            logger.log(Level.WARNING, "Error closing HttpClient", e);
+        }
```

Comment:
This sets up an HTTP client to do this user id resolution, and then closes it. `makeApiPost()` creates its own HTTP client. Is that okay? I don't know if these HTTP clients are lightweight or not.